ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -30,6 +30,25 @@ enum Progress {
   Redirecting
 }
 
+export enum Kernels {
+  R,
+  Python3,
+}
+
+const rNotebookFileContent = '{ "cells": [ { "cell_type": "code", "execution_count": null, \
```

Comment:
Please define this as an object, then use JSON.stringify(...) to turn it into a string. Else you're needlessly skipping validation of this JSON content.

Reply:
Will do!